### PR TITLE
Feat/obfuscation

### PR DESCRIPTION
**Pull request by samuellye98**
Obfuscation function

### CODE REVIEW

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=26

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=26

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=26

File: src/components/SharePage/obfuscateFields.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const obfuscateFields = [
+  { title: "Pass Type", key: "pass.type" },
+  {
```
Comment (bennchong) [inline src/components/SharePage/obfuscateFields.ts]: Why is this styled differently!!

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=26

**Pull request merged**